docs/website/contents/for-developers/Benchmarks.md

```diff
@@ -1,11 +1,22 @@
 # Consensus benchmarks
 
 We are in the process of adding component level microbenchmarks for Consensus.
+
+We check for regressions in performance on CI.
+
+## Mempool Benchmark
+
 We started with microbenchmarks for adding transactions to the mempool. The
 mempool benchmarks can be run using the following command.
 
 ```sh
 cabal new-run ouroboros-consensus:mempool-bench
 ```
 
-We check for regressions in performance on CI. We might publish benchmark results in this site shortly.
+## ChainSync Client Benchmark
+
+During the work on Ouroboros Genesis, we added a ChainSync benchmark which uses test blocks:
```

Comment:
I don't think this work was done as part of the Genesis SoW FWIW (IIRC #823 was prompted by Nick working with the ChainSync client in #525 and then being motivated to do some refactorings for maintainability).